.sass-lint.yml

```diff
@@ -8,7 +8,10 @@ options:
 
 files:
   include: '+(addon|app|tests)/**/styles/**/*.s+(a|c)ss'
-  ignore: '**/dummy/app/styles//**/*.s+(a|c)ss'
+  ignore:
+    - '**/dummy/app/styles//**/*.s+(a|c)ss'
+    - '**/_flex.scss'
```

Comment:
Humm let me give it another try. I will show you the end result to make sure i'm not breaking anything :)